.devcontainer/devcontainer.json

```diff
@@ -6,7 +6,12 @@
 	"image": "mcr.microsoft.com/devcontainers/dotnet",
 	"features": {
 		"ghcr.io/devcontainers/features/powershell:1": {},
-		"ghcr.io/devcontainers/features/dotnet:2": {}
+		"ghcr.io/devcontainers/features/dotnet:2": {
```

Comment:
I've been meaning to do this in like forever, but I keep forgetting!

@paule96 Do you mind creating a new pull request that only updates the devcontainer setup?